common/build/eslint-config-fluid/base.js

```diff
@@ -349,6 +349,24 @@ module.exports = {
 		"import/resolver": {
 			typescript: {
 				extensions: [".ts", ".tsx", ".d.ts", ".js", ".jsx"],
+				conditionNames: [
```

Comment:
once I have some more confidence these are all the changes we need to make to the base config, I'll split this into a different PR.